From b3ef5627c6339dd09a9a352e9b5960f6fd829159 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 9 Feb 2018 11:34:27 -0800
Subject: [PATCH] TOOLS-1981 avoid unexpected changes to package.json in Joyent
 repos Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved by: Cody
 Peter Mello <cody.mello@joyent.com>

---
 tools/mk/Makefile.deps | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/mk/Makefile.deps b/tools/mk/Makefile.deps
index 7419253..91f8346 100644
--- a/tools/mk/Makefile.deps
+++ b/tools/mk/Makefile.deps
@@ -66,14 +66,14 @@ $(ESLINT_EXEC): package.json | $(NPM_EXEC)
 	ESLINT_VER=$$($(NODE) -e 'console.log(require("./package.json").devDependencies["eslint"] || "")') && \
 	    ESLINT_JOY_VER=$$($(NODE) -e 'console.log(require("./package.json").devDependencies["eslint-plugin-joyent"] || "")') && \
 	    [[ -n $$ESLINT_VER && -n $$ESLINT_JOY_VER ]] && \
-	    $(NPM) install eslint@$$ESLINT_VER eslint-plugin-joyent@$$ESLINT_JOY_VER && \
+	    $(NPM) install --no-save eslint@$$ESLINT_VER eslint-plugin-joyent@$$ESLINT_JOY_VER && \
 	    touch $(ESLINT_EXEC)
 else
 $(ESLINT_EXEC): package.json
 	ESLINT_VER=$$(node -e 'console.log(require("./package.json").devDependencies["eslint"] || "")') && \
 	    ESLINT_JOY_VER=$$(node -e 'console.log(require("./package.json").devDependencies["eslint-plugin-joyent"] || "")') && \
 	    [[ -n $$ESLINT_VER && -n $$ESLINT_JOY_VER ]] && \
-	    npm install eslint@$$ESLINT_VER eslint-plugin-joyent@$$ESLINT_JOY_VER && \
+	    npm install --no-save eslint@$$ESLINT_VER eslint-plugin-joyent@$$ESLINT_JOY_VER && \
 	    touch $(ESLINT_EXEC)
 endif
 
-- 
2.21.0

